lib/src/cpp/transport_descriptor/TransportDescriptor.cpp

```diff
@@ -614,7 +636,20 @@ void set_kind(
         const std::string& transport_descriptor_id,
         const std::string& kind)
 {
-    throw Unsupported("Unsupported");
+    // Create XML manager and initialize the document
+    utils::XMLManager manager(xml_file, true);
+
+    // Obtain base node position
+    initialize_namespace(manager, transport_descriptor_id, true);
+
+    // Obtain kind node
+    manager.get_node(utils::tag::TRANSPORT_KIND, true);
```

Comment:
Please, apply the same suggestion as in the previous PR and also to the next PR too.